commands/staff/rm.js

```diff
@@ -1,12 +1,11 @@
 const { fn, emojis } = require("../../config")
-const db = require("quick.db")
-const { players } = require("../../db.js")
+// const { players } = require("../../db.js")
```

Comment:
???

Reply:
same players is not used its waste of memory

Reply:
since commented data commented players, this will be reverted.

Reply:
done in [af2d2a1](https://github.com/wwosimulation/Narrator-bot/pull/85/commits/af2d2a15f6126e5001234f53d7f4be1e0ec06bbe)